### PR TITLE
docs: proposal for listing maintainers

### PR DESCRIPTION
**Pull request by danielpeintner**
fixes https://github.com/eclipse/thingweb.node-wot/issues/986

### CODE REVIEW

**Comment by egekorkan** [comment]
Looks good to me and we can do the same for the other packages if we are fine with this.

**Comment by relu91** [comment]
Yes, greenlight also from my side.

**Comment by JKRhb** [comment]
I was wondering if we should also add a `CODEOWNERS` file to reflect the responsibilities for the different packages here on GitHub? If so, I could make a proposal for that in a follow-up PR

> Note2: We currently list the people like this `@danielpeintner`. I think this is not going to become a link.. especially not on NPM. Do we want to add a link to the GitHub account as well or is it fine as is.

I was thinking the same, wrapping the `@person`s into links sounds like a good idea (even though it is a bit tedious).

File: packages/binding-coap/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 W3C WoT Binding Template specification for CoAP can be found [here](https://w3c.github.io/wot-binding-templates/bindings/protocols/coap/index.html).
 
+Current Maintainer(s): @JKRhb
```
Comment (JKRhb) [inline packages/binding-coap/README.md:5]:
```suggestion
Current Maintainer(s): [@JKRhb](https://github.com/JKRhb)
```

File: packages/binding-http/README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 W3C WoT Binding Template specification for HTTP can be found [here](https://w3c.github.io/wot-binding-templates/bindings/protocols/http/index.html).
 
+Current Maintainer(s): @relu91 @danielpeintner
```
Comment (JKRhb) [inline packages/binding-http/README.md:5]:
```suggestion
Current Maintainer(s): [@relu91](https://github.com/relu91) [@danielpeintner](https://github.com/danielpeintner)
```

File: packages/binding-mbus/README.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ W3C Web of Things (WoT) Protocol Binding for [Meter Bus](https://en.wikipedia.or
 This package uses [node-mbus](https://www.npmjs.com/package/node-mbus) as a low level client for M-Bus TCP.
 This implementation only supports reading data (for WoT Consumer applications).
 
+Current Maintainer(s): @LennyHEVS
```
Comment (JKRhb) [inline packages/binding-mbus/README.md:9]:
```suggestion
Current Maintainer(s): [@LennyHEVS](https://github.com/LennyHEVS)
```

File: packages/binding-modbus/README.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ W3C Web of Things (WoT) Protocol Binding for [Modbus](https://en.wikipedia.org/w
 This package uses [modbus-serial](https://www.npmjs.com/package/modbus-serial) as a low level client for Modbus TCP.
 W3C WoT Binding Template for Modbus can be found [here](https://w3c.github.io/wot-binding-templates/bindings/protocols/modbus/index.html).
 
+Current Maintainer(s): @relu91 @fillobotto
```
Comment (JKRhb) [inline packages/binding-modbus/README.md:9]:
```suggestion
Current Maintainer(s): [@relu91](https://github.com/relu91)  [@fillobotto](https://github.com/fillobotto) 
```

File: packages/binding-netconf/README.md [hunk 5]
```diff
@@ -2,6 +2,8 @@
 
 W3C Web of Things (WoT) Protocol Binding for NETCONF [RFC6241](https://tools.ietf.org/html/rfc6241)
 
+Current Maintainer(s): @lukesmolo
```
Comment (JKRhb) [inline packages/binding-netconf/README.md:5]:
```suggestion
Current Maintainer(s): [@lukesmolo](https://github.com/lukesmolo)
```

File: packages/binding-opcua/README.md [hunk 6]
```diff
@@ -3,6 +3,8 @@
 W3C Web of Things (WoT) Protocol Binding for OPC UA.
 This package uses [nodep-opcua](https://www.npmjs.com/package/node-opcua) as a low-level client for OPCUA over TCP.
 
+Current Maintainer(s): @erossignon
```
Comment (JKRhb) [inline packages/binding-opcua/README.md:6]:
```suggestion
Current Maintainer(s): [@erossignon](https://github.com/erossignon)
```

File: packages/binding-websockets/README.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Websocket Protocol Binding of node-wot
+
+Current Maintainer(s): @miguelrk
```
Comment (JKRhb) [inline packages/binding-websockets/README.md:3]:
```suggestion
Current Maintainer(s): [@miguelrk](https://github.com/miguelrk)
```

File: packages/td-tools/README.md [hunk 8]
```diff
@@ -1,5 +1,7 @@
 # TD (Thing Description) tools of node-wot
 
+Current Maintainer(s): @danielpeintner @relu91
```
Comment (JKRhb) [inline packages/td-tools/README.md:3]:
```suggestion
Current Maintainer(s): [@danielpeintner](https://github.com/danielpeintner) [@relu91](https://github.com/relu91)
```

File: packages/core/README.md [hunk 9]
```diff
@@ -8,6 +8,8 @@ The core of node-wot is the entry point allowing to attach dedicated bindings su
 
 or to create user-specific bindings.
 
+Current Maintainer(s): @danielpeintner @relu91 @JKRhb
```
Comment (JKRhb) [inline packages/core/README.md:11]:
```suggestion
Current Maintainer(s): [@danielpeintner](https://github.com/danielpeintner) [@relu91](https://github.com/relu91) [@JKRhb](https://github.com/JKRhb)
```

File: packages/cli/README.md [hunk 10]
```diff
@@ -1,5 +1,7 @@
 # Command-line interface (CLI) of node-wot
 
+Current Maintainer(s): @relu91 @danielpeintner @mkovatsc
```
Comment (JKRhb) [inline packages/cli/README.md:3]:
```suggestion
Current Maintainer(s): [@relu91](https://github.com/relu91) [@danielpeintner](https://github.com/danielpeintner) [@mkovatsc](https://github.com/mkovatsc)
```

File: packages/browser-bundle/README.md [hunk 11]
```diff
@@ -2,6 +2,8 @@
 
 Bundle to run node-wot as browser-side library. Note, this will only include the node-wot in client mode with limited binding support.
 
+Current Maintainer(s): @relu91 @danielpeintner
```
Comment (JKRhb) [inline packages/browser-bundle/README.md:5]:
```suggestion
Current Maintainer(s): [@relu91](https://github.com/relu91) [@danielpeintner](https://github.com/danielpeintner)
```

File: packages/binding-mqtt/README.md [hunk 12]
```diff
@@ -4,6 +4,8 @@ W3C Web of Things (WoT) Protocol Binding for [MQTT](https://en.wikipedia.org/wik
 This package uses [mqtt](https://www.npmjs.com/package/mqtt) as a low level library for MQTT.
 W3C WoT Binding Template for MQTT can be found [here](https://w3c.github.io/wot-binding-templates/bindings/protocols/mqtt/index.html).
 
+Current Maintainer(s): @egekorkan @hasbel @sebastiankb @hasanheroglu
```
Comment (JKRhb) [inline packages/binding-mqtt/README.md:7]:
```suggestion
Current Maintainer(s): [@egekorkan](https://github.com/egekorkan) [@hasbel](https://github.com/hasbel) [@sebastiankb](https://github.com/sebastiankb) [@hasanheroglu](https://github.com/hasanheroglu)
```

**Review by JKRhb — COMMENTED** [review]
Took the freedom of adding the link wrappers as suggestions in the comments below:

**Comment by danielpeintner** [comment]
Thanks @JKRhb for your updates.

I think this PR is now ready for review

**Review by JKRhb — APPROVED** [review]

**Review by relu91 — APPROVED** [review]
LGTM

**Comment by danielpeintner** [comment]
Thanks all!

**Pull request merged**